packages/fast-usdc/test/exos/advancer.test.ts

```diff
@@ -162,7 +189,7 @@ test('updates status to ADVANCED in happy path', async t => {
 
   t.deepEqual(inspectLogs(0), [
     'Advance transfer fulfilled',
-    '{"amount":{"brand":"[Alleged: USDC brand]","value":"[150000000n]"},"destination":{"chainId":"osmosis-1","encoding":"bech32","value":"osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men"},"result":"[undefined]"}',
+    '{"amount":{"brand":"[Alleged: USDC brand]","value":"[149999899n]"},"destination":{"chainId":"osmosis-1","encoding":"bech32","value":"osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men"},"result":"[undefined]"}',
```

Comment:
suggestion: add a test to `fees.tests.ts` using the numbers in this exo test such as `149999899n`.

Then I wouldn't have to do arithmetic to review this PR.

Reply:
Good suggestion. See [ac61b3666e](https://github.com/Agoric/agoric-sdk/pull/10518/commits/ac61b3666ecf7679264174b91580f92f54d68f03)

Reply:
Perhaps better without the underscores. LMK